frontend/components/ClubEditPage/AdvisorCard.tsx

```diff
@@ -64,13 +94,19 @@ export default function AdvisorCard({ club, schools }: Props): ReactElement {
           )}
           fields={fields}
         />
+        {SITE_NAME === 'Hub@Penn' && (
```

Comment:
Ideally, use `SITE_ID` instead of `SITE_NAME`, just for consistency purposes. If we ever change up this check in the future, we can find all instances in the codebase by grepping for "SITE_ID".